old-tests/tests_onhold/4_server-side/ajax.js

```diff
@@ -532,7 +532,7 @@ $(document).ready( function () {
 		},
 		function () {
 			console.log( result );
-			return $.isArray( result[0] ) && result[0].length === 35;
+			return Array.isArray( result[0] ) && result[0].length === 35;
```

Comment:
Not sure if polyfill from `core.internal.js` is available in test files, or if its required?